plasma_framework/test/src/exits/utils/TxFinalization.test.js

```diff
@@ -0,0 +1,202 @@
+const TxFinalization = artifacts.require('TxFinalizationWrapper');
+const PlasmaFramework = artifacts.require('SpyPlasmaFrameworkForExitGame');
+
+const { expect } = require('chai');
+const { constants } = require('openzeppelin-test-helpers');
+
+const { buildTxPos } = require('../../../helpers/positions.js');
+const { MerkleTree } = require('../../../helpers/merkle.js');
+const { sign } = require('../../../helpers/sign.js');
+const { PROTOCOL, EMPTY_BYTES } = require('../../../helpers/constants.js');
+
+contract('TxFinalization', ([richFather]) => {
+    const TEST_BLOCK_NUM = 1000;
+    const alicePrivateKey = '0x7151e5dab6f8e95b5436515b83f423c4df64fe4c6149f864daa209b26adb10ca';
+    let alice;
+
+    before('setup test contract', async () => {
+        this.test = await TxFinalization.new();
+    });
+
+    before('setup alice account with custom private key', async () => {
+        const password = 'password1234';
+        alice = await web3.eth.personal.importRawKey(alicePrivateKey, password);
+        alice = web3.utils.toChecksumAddress(alice);
+        web3.eth.personal.unlockAccount(alice, password, 3600);
+        web3.eth.sendTransaction({
+            to: alice,
+            from: richFather,
+            value: web3.utils.toWei('1', 'ether'),
+        });
+    });
+
+    describe('isStandardFinalized', () => {
+        beforeEach(async () => {
+            this.framework = await PlasmaFramework.new(0, 0, 0);
+        });
+
+        it('should revert when invalid protocol value provided', async () => {
+            const invalidProtocol = 199;
+
+            const verifier = await this.test.getVerifier(
+                constants.ZERO_ADDRESS,
+                invalidProtocol,
+                EMPTY_BYTES,
+                0,
+                EMPTY_BYTES,
+                EMPTY_BYTES,
+                constants.ZERO_ADDRESS,
+            );
+
+            const errorMsg = 'Returned error: VM Exception while processing transaction: revert invalid protocol value';
+            await this.test.isStandardFinalized(verifier)
```

Comment:
This is something mysterious to me actually. I don't know why, but that does not work @__@"
I will try again if I can get out of this or understand the reason why it doesn't work.

I've tried to use `require(false, "xxx")` instead of revert and still failing.

Reply:
ok......then it works now @"@
Perhaps I did sth stupid yesterday......